paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/ui/GooglePayButtonTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.paymentsheet.MainActivity
+import org.junit.Rule
+import org.junit.Test
+
+class GooglePayButtonTest {
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+    @Test
+    fun handlesPressWhenEnabled() {
+        val testTag = "google-pay-button"
```

Comment:
Should this use `GooglePayButton.TEST_TAG` instead?

Reply:
Yes!